handlers/whatsapp/whatsapp.go

```diff
@@ -319,18 +319,6 @@ func (h *handler) BuildAttachmentRequest(ctx context.Context, b courier.Backend,
 
 var _ courier.AttachmentRequestBuilder = (*handler)(nil)
 
-var waStatusMapping = map[string]courier.MsgStatus{
-	"sending":   courier.MsgStatusWired,
-	"sent":      courier.MsgStatusSent,
-	"delivered": courier.MsgStatusDelivered,
-	"read":      courier.MsgStatusDelivered,
-	"failed":    courier.MsgStatusFailed,
-}
-
-var waIgnoreStatuses = map[string]bool{
-	"deleted": true,
-}
```

Comment:
I'm inclined to say we don't touch this handler and instead rename it to `whatsapp_legacy` and let it die

Reply:
I thought so but someone started changing that a few days ago!!

Reply:
Haha fair point yeah I should have left it - when you're finished with this pr let's rename this to `whatsapp_legacy`... and maybe also `facebook` to `facebook_legacy` and `facebookapp` to... `meta` ??